frontend/app/components/new-tab-indicator.tsx

```diff
@@ -0,0 +1,10 @@
+import type { ComponentProps, ReactNode } from 'react';
+
+export interface NewTabIndicatorProps extends ComponentProps<'span'> {
+  children: ReactNode;
+}
+
+export function NewTabIndicator(props: NewTabIndicatorProps) {
+  const { children } = props;
+  return <span className="sr-only">{children}</span>;
+}
```

Comment:
Handle the content here. `&#32` is important to ensure the content's text is seperated for the `opens in a new tab` text.

```suggestion
export function NewTabIndicator({className, ...props}: NewTabIndicatorProps) {
  const { t } = useTranslation("gcweb");
  return <span className={cn("sr-only", className)} ...props>&#32;({t('gcweb:screen-reader.newtab')})</span>;
}
```

Reply:
Just use a regular space, and consider adding a comment for future devs so it isn't removed..

```
// note: leading space is required for screen readers to correctly identify word boundaries
<span ..> ({t('..')})</span>
```

something like that

Reply:
@gregory-j-baker I think to be safe we should explicitly set it `&#32`. I don't want Prettier to format the file and remove it.

Reply:
@sebastien-comeau always has to disagree 🏃🏻